include/glow/Importer/CommonOperatorLoader.h

```diff
@@ -88,16 +100,16 @@ inline llvm::Error loadWeight(const onnxTensorDescriptorV1 &in, Tensor *T) {
         TH.raw(i) = data[i];
       }
     } else if (in.dataType == ONNXIFI_DATATYPE_INT32) {
-      T->reset(ElemKind::Int32ITy, dims);
+      result.t->reset(ElemKind::Int32ITy, dims);
 
-      auto TH = T->getHandle<int32_t>();
+      auto TH = result.t->getHandle<int32_t>();
       int32_t *data = (int32_t *)in.buffer;
       for (size_t i = 0; i < TH.size(); ++i) {
         TH.raw(i) = data[i];
       }
     } else if (in.dataType == ONNXIFI_DATATYPE_UINT8) {
-      T->reset(ElemKind::Int8QTy, dims, 1.0, 0);
-      auto TH = T->getHandle<int8_t>();
+      result.t->reset(ElemKind::Int8QTy, dims, 1.0, 0);
+      auto TH = result.t->getHandle<int8_t>();
       uint8_t *data = (uint8_t *)in.buffer;
       for (size_t i = 0; i < TH.size(); ++i) {
         constexpr uint8_t OFFSETSHIFT = 128;
```

Comment:
what's the reason OFFSETSHIFT is defined again?

Reply:
Was wondering that too, it's from a previous commit, I'll remove it